docs/ENVS.md

```diff
@@ -144,6 +145,16 @@ By default, the app has generic favicon. You can override this behavior by provi
 
 &nbsp;
 
+### Meta
+
+| Variable | Type| Description | Compulsoriness  | Default value | Example value |
+| --- | --- | --- | --- | --- | --- |
+| NEXT_PUBLIC_PROMOTE_BLOCKSCOUT_IN_TITLE | `boolean` | Set to `true` to promote Blockscout in page titles | - | `true` | `true` |
+| NEXT_PUBLIC_OG_DESCRIPTION | `string` | OG description for the homepage | - | - | `Blockscout is the #1 open-source blockchain explorer available today. 100+ chains and counting rely on Blockscout data availability, APIs, and ecosystem tools to support their networks.` |
+| NEXT_PUBLIC_OG_IMAGE_URL | `string` | OG image url for homepage and root pages. Minimum image size is 200 x 20 pixels; maximum supported file size is 8 MB; 1.91:1 aspect ratio; supported formats: image/jpeg, image/gif, image/png | - | `static/og_placeholder.png` | `https://placekitten.com/1200/600` |
```

Comment:
Where does the requirement of maximum size and aspect ratio come from? Why in the example the aspect ratio is `2` and not `1.91:1`?